drivers/hmis/app/graphql/mutations/bulk_assign_service.rb

```diff
@@ -53,7 +53,8 @@ def resolve(input:)
             access_denied! unless can_enroll_clients
             raise 'bulk service assignment generated invalid enrollment' unless enrollment.valid?
 
-            # TODO(#187288068) warn about overlapping enrollments using Hmis::Hud::Validators::EnrollmentValidator.validate_entry_date
+            entry_date_errors = Hmis::Hud::Validators::EnrollmentValidator.validate_entry_date(enrollment)
+            raise HmisErrors::ApiError, entry_date_errors.first.full_message unless entry_date_errors.empty?
```

Comment:
This has a pretty unintuitive API, you have to pass `display_message` like [this](https://github.com/greenriver/hmis-warehouse/blob/fb5fe917564ac0ea99e453c6b451f035731cce39/drivers/hmis/app/graphql/mutations/bulk_assign_service.rb#L105) if you want the message to actually be visible to the user, which we do in this case. You can mimic production behavior(sort of) by adjusting this code: https://github.com/greenriver/hmis-warehouse/blob/fb5fe917564ac0ea99e453c6b451f035731cce39/drivers/hmis/app/controllers/hmis/graphql_controller.rb#L103

Reply:
Ah thank you! It looks like there is already a helper `error_out` that does this, so I changed it to use that.

<img width="930" alt="Screenshot 2024-04-03 at 5 00 31 PM" src="https://github.com/greenriver/hmis-warehouse/assets/16002775/6b1686fc-a92d-441b-9818-fbe0c4804616">

Reply:
nice, thanks!